src/Core/src/Handlers/Editor/EditorHandler.Android.cs

```diff
@@ -60,8 +61,10 @@ protected override void DisconnectHandler(AppCompatEditText platformView)
 			_set = false;
 		}
 
-		public static void MapBackground(IEditorHandler handler, IEditor editor) =>
+		public static void MapBackground(IEditorHandler handler, IEditor editor)
```

Comment:
Because this file doesn't have any functional changes can we leave this file unchanged?